src/CollapsibleBlock/CollapsibleBlock.tsx

```diff
@@ -0,0 +1,26 @@
+import { NodeViewContent, NodeViewWrapper } from '@tiptap/react';
+import './styles.css';
+
+export const CollapsibleBlock = (props: any) => {
```

Comment:
Define a TypeScript type for props.

Reply:
Unfortunately typing TipTap decorator nodes is not straight forward as they use. I have added a comment and some extra types to prevent having runtime errors